test/spec/algoliasearch.helper/incorrectFacetDefinition.js

```diff
@@ -1,38 +1,31 @@
 'use strict';
 
-var test = require('tape');
 var algoliasearchHelper = require('../../../index');
 
 var _ = require('lodash');
 
 var fakeClient = {};
 
-test('Conjuctive facet should be declared to be refined', function(t) {
+test('Conjuctive facet should be declared to be refined', function() {
   var h = algoliasearchHelper(fakeClient, '', {});
 
-  t.throws(_.bind(h.addRefine, h, 'undeclaredFacet', 'value'), 'Adding a facet refinement should not be possible');
-  t.throws(_.bind(h.removeRefine, h, 'undeclaredFacet', 'value'), 'Remove a facet refinement should not be possible');
-  t.throws(_.bind(h.isRefined, h, 'undeclaredFacet', 'value'), 'Checking if a facet is refined should not be possible');
-
-  t.end();
+  expect(_.bind(h.addRefine, h, 'undeclaredFacet', 'value')).toThrow();
```

Comment:
Do we want to track which errors are thrown?

Reply:
See https://github.com/algolia/algoliasearch-helper-js/pull/660#discussion_r278628453